bin-dir/system-test-script.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e
```

Comment:
use `set -euo pipefail`, `-u` will fail on undefined variables, and `pipefail` will fail if a process dies after a pipe. You don't have any pipes here yet but it's good practice.

Reply:
Sure